ANMN/burst_averaged_product/burstproduct/burst_average_product.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+from generate_netcdf_att import *
+from math import isnan
+from netCDF4 import Dataset, num2date, date2num
+import numpy as np
+import os
+import pandas as pd
+import re
+import shutil
+import subprocess
+import tempfile
+import time
+
+"""TODO
+- cleaning get_input_file_rel_path, handle error
+- logging
+
+- for CTD file
+http://thredds.aodn.org.au/thredds/catalog/IMOS/ANMN/NRS/NRSKAI/CTD_timeseries/catalog.html
+- remove VOLT, TIMESERIES, NOMINAL_DEPTH????
+- add all variables to existing conf file. Should work also for WQM file
+- refactore 1 or 2 WQM function as CTD will be very similar
+"""
+
+
+def get_input_file_rel_path(input_netcdf_file_path):
```

Comment:
Seems unnecessarily complicated to use a shell subprocess to call another Python script! The guts or `ANMN/common/dest_path.py` is actually a class defined in `lib/python/file_classifier.py`, so you can just do this:
```python
from file_classifier import MooringFileClassifier

file_path = MooringFileClassifier.dest_path(input_netcdf_file_path)
```